modules/fastcdr/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://github.com/eProsima/Fast-CDR",
+    "maintainers": [],
```

Comment:
Would you be willing to add yourself as a maintainer for this BCR entry?

Reply:
add myself as a maintainer at https://github.com/bazelbuild/bazel-central-registry/pull/1345/commits/3ae2520d4ec26e6ff4059a021dd699177cf0e1c4